stable/sumologic-fluentd/Chart.yaml

```diff
@@ -1,5 +1,6 @@
 name: sumologic-fluentd
-version: 0.3.1
+version: 0.3.2
```

Comment:
You're adding a feature here, so please bump the minor version.

Reply:
https://github.com/kubernetes/charts/pull/4539/commits/5275d04f81bf83deb436efd9b93f87eba36be58b